tests/node.js

```diff
@@ -122,24 +121,27 @@ function runTest(fileName, type, binary, wat) {
   // should not block testing
   promises.push(fs.writeFile(watPath, wat));
 
-  const context = new TestContext({
-    fileName: fileNamePath, // set the fileName
-    reporter, // use verbose reporter
-    binary, // pass the binary to get function names
-  });
   const wasi = new WASI({
     args: [],
```

Comment:
```suggestion
    args: args: process.argv,
```

Reply:
I believe we should actually make this configurable via `testname.spec.wasi.js` which would be shaped something like this:

```js
module.exports = {
  env: {},
  argv: [],
  preopens: {}
}
```

Do you think this is a better idea?

Reply:
env / argv just pass from node.js environment. `preopens` could be configured via cli argument

Reply:
Why not both? And a `json` configuration would probably not work well, because we will want access to `process.env` likely.

Reply:
It's up to you. But if you want support many ways you should decide overriding order or union instead. So it becomes more complicated

Reply:
I went with `--preopens dir1=dir1,dir2=dir2` etc.

And `{testname}.spec.wasi.js` will override the default configuration completely.